server.ts

```diff
@@ -49,11 +58,9 @@ app.get("/", (req, res) => {
   } else res.redirect("/survey");
 });
 
-// Test URL: https://raw.githubusercontent.com/Watts-Lab/surveyor/main/surveys/CRT.csv
-// e.g. http://localhost:4000/s/?url=https://raw.githubusercontent.com/Watts-Lab/surveyor/main/surveys/CRT.csv&name=Mark
-app.get("/s/", async (req, res) => {
+const getsurvey = async (urlstr: string, req: Request<{}>, res: Response<any>) =>  {
```

Comment:
So perhaps, given my other comment below, this could take a `URL`, a `query` object as well as `req` and `res`. Or just call the URL from the `query` object to simplify things?